### PR TITLE
[5.0][bug] TinyMCE Setting linebreak with doesn´t work

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #42224 .

### Summary of Changes

Use correctly the [`newline_behavior`](https://www.tiny.cloud/docs/tinymce/6/content-behavior-options/#newline_behavior) parameter

### Testing Instructions

Follow the issue description

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 926b25281c5069069ab8928b9ea85d0e0677f922<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42227">issues.joomla.org/tracker/joomla-cms/42227</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 926b25281c5069069ab8928b9ea85d0e0677f922<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42227">issues.joomla.org/tracker/joomla-cms/42227</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42227">issues.joomla.org/tracker/joomla-cms/42227</a>.</sub>

File: plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php [hunk 1]
```diff
@@ -464,10 +464,10 @@ public function display(string $name, string $content = '', array $attributes =
 
         if ($levelParams->get('newlines')) {
             // Break
-            $scriptOptions['force_br_newlines'] = true;
+            $scriptOptions['newline_behavior'] = 'invert';
         } else {
             // Paragraph
-            $scriptOptions['force_br_newlines'] = false;
+            $scriptOptions['newline_behavior'] = 'default';
```
Comment (Quy) [inline plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php]:
```suggestion
            $scriptOptions['newline_behavior']  = 'default';
```

**Comment by bembelimen** [comment]
Thanks

**Comment by BPBlueprint** [comment]
I have tested this item ✅ successfully. Thanks!

**Pull request merged**